### PR TITLE
Making the QoS section a bit less technical

### LINKED ISSUE #748: Making the QoS section a bit less technical
Re: [jamulussoftware/jamuluswebsite] Reinhardwh de v4 (PR #747)
@ann0see

Hi, I received an email from GitHub on this topic and can't find where to reply so I opened an Issue.
I modified two files to address this issue: (.diff provided for quick inspection, or cut and paste, or patch)

../wiki/en/Include-QOS-Windows.md.diff
```
--- Include-QOS-Windows.md	2022-05-21 13:46:54.000000000 -0700
+++ Include-QOS-Windows.md	2022-06-10 12:34:39.699702688 -0700
@@ -3,11 +3,11 @@
 
 # Quality of Service (QoS)
 
-Jamulus uses DSCP/CS4 opportunistically to deal with buffer bloat, and uses a default DSCP/CS4 value of 128 (or 0x80). This is compatible with IPv4 and IPv6. Other values can be set using the -Q option, eg -Q [0..255] (where 0 disables QoS). If you want to explore the effect of non-default settings, see RFC4594. However, most people will have no need to do this.
+Jamulus uses DSCP/CS4 to opportunistically deal with buffer bloat. Setting the DS field byte to 128 (or 0x80) in a IPv4 or IPv6 packet header selects DSCP/CS4. Other byte values can be set using the -Q option, eg -Q [0..255] (where 0 cripples QoS). If you want to explore the effect of non-default settings, see RFC4594.
 
 ## Use of QoS on Windows
 
-Jamulus's QoS settings (including the default) have no effect on Windows because the operating system ignores them. To enable it, you must follow these instructions. Note also that you may need to repeat this procedure every time Jamulus is updated.
+Jamulus' QoS settings (including the default) are set to 0 on recent Windows and Apple because of the operating system. To use QoS on Windows, follow these instructions. Note that you may need to repeat this procedure every time Jamulus is updated.
 
 In Search box beside Start menu Type: Local Group Policy Editor (enter)<br>
 In new window, (click) on the menu icon to display the Actions third panel<br>
```

../wiki/en/Include-Shared-Commands.md.diff
```
--- Include-Shared-Commands.md	2022-05-21 13:46:54.000000000 -0700
+++ Include-Shared-Commands.md	2022-06-10 11:23:02.191837116 -0700
@@ -5,7 +5,7 @@
 - `-i` or `--inifile`   Set location of initialization file (overrides default) 
 - `-n` or `--nogui`    Disable GUI (for use in headless mode)                      
 - `-p` or `--port`    Sets the local UDP port number. Default is 22124 
-- `-Q` or `--qos`     Sets the quality of service DSCP value. Default is 128. Disable with 0. QoS is ignored by Windows. To enable it, see
+- `-Q` or `--qos`     Sets the quality of service DS Field byte. Default is 128 (DSCP/CS4). QoS is ignored by Windows. To enable it, see
 section below
 - `-t` or `--notranslation`   Disable UI language translations
 -  `-6` or `--enableipv6`      Enable IPv6 addressing (IPv4 is always enabled)
```
[Include-QOS-Windows.new.md](https://github.com/jamulussoftware/jamuluswebsite/files/8882077/Include-QOS-Windows.new.md)
[Include-Shared-Commands.new.md](https://github.com/jamulussoftware/jamuluswebsite/files/8882080/Include-Shared-Commands.new.md)

### PR DESCRIPTION
**Pull request by ghost**
**Short description of changes**
Two files in the QoS section are re-worded to be simple and concise.
**Context: Fixes an issue? Related issues**
 Hopefully makes the English text easier to translate into other languages.
**Status of this Pull Request**

**What is missing until this pull request can be merged?**
Needs to to reviewed

**Does this need translation?**

Yes

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

File: wiki/en/Include-QOS-Windows.md [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
+Jamulus uses DSCP/CS4 to opportunistically deal with buffer bloat. Setting the DS field byte to 128 (or 0x80) in a IPv4 or IPv6 packet header selects DSCP/CS4. Other byte values can be set using the -Q option, e.g. -Q [0..255] (where 0 cripples QoS). If you want to explore the effect of non-default settings, see RFC4594. However, most people will have no need to do this.
 
 ## Use of QoS on Windows
 
-Jamulus's QoS settings (including the default) have no effect on Windows because the operating system ignores them. To enable it, you must follow these instructions. Note also that you may need to repeat this procedure every time Jamulus is updated.
+Jamulus' QoS settings (including the default) are set to 0 on recent Windows and Apple because of the operating system. To use QoS on Windows, follow these instructions. Note that you may need to repeat this procedure every time Jamulus is updated.
```
Comment (ann0see) [inline wiki/en/Include-QOS-Windows.md]:
```suggestion
Jamulus' QoS settings (including the default) are set to 0 on recent Windows and macOS because of the operating system. To use QoS on Windows, follow these instructions. Note that you may need to repeat this procedure every time Jamulus is updated.
```
Is macOS now also un-changable? Why did you add it here?

File: wiki/en/Include-Shared-Commands.md [hunk 2]
```diff
@@ -2,11 +2,10 @@
...
-- `-p` or `--port`    Sets the local UDP port number. Default is 22124 
-- `-Q` or `--qos`     Sets the quality of service DSCP value. Default is 128. Disable with 0. QoS is ignored by Windows. To enable it, see
-section below
+- `-p` or `--port`     Sets the local UDP port number. Default is 22124 
+- `-Q` or `--qos`      Sets the quality of service DS Field byte. Default is 128 (DSCP/CS4). QoS is ignored by Windows. To enable it, see section below.
 - `-t` or `--notranslation`   Disable UI language translations
--  `-6` or `--enableipv6`      Enable IPv6 addressing (IPv4 is always enabled)
-- `-v` or `--version`   Output version information and exit 
+- `-6` or `--enableipv6`      Enable IPv6 addressing (IPv4 is always enabled)
+- `-v` or `--version`  Output version information and exit
```
Comment (ann0see) [inline wiki/en/Include-Shared-Commands.md:11]: Spacing seems a bit odd here.

File: wiki/en/Include-QOS-Windows.md [hunk 3]
```diff
@@ -3,13 +3,13 @@
 
 # Quality of Service (QoS)
 
-Jamulus uses DSCP/CS4 opportunistically to deal with buffer bloat, and uses a default DSCP/CS4 value of 128 (or 0x80). This is compatible with IPv4 and IPv6. Other values can be set using the -Q option, eg -Q [0..255] (where 0 disables QoS). If you want to explore the effect of non-default settings, see RFC4594. However, most people will have no need to do this.
+Jamulus uses DSCP/CS4 to opportunistically deal with buffer bloat. Setting the DS field byte to 128 (or 0x80) in a IPv4 or IPv6 packet header selects DSCP/CS4. Other byte values can be set using the -Q option, e.g. -Q [0..255] (where 0 cripples QoS). If you want to explore the effect of non-default settings, see RFC4594. However, most people will have no need to do this.
```
Comment (ann0see) [inline wiki/en/Include-QOS-Windows.md]: I'd link DSCP/CS4 to a page describing it (i.e Wikipedia). Something not technical to allow users to decide what they do. Same with Buffer bloat. I think a site is already linked somewhere else.

Comment (ghost) [reply wiki/en/Include-QOS-Windows.md]: Links will eventually get broken. I think sincere users will easily find the information by searching the internet.
Note that "most people will have no need to do this" because Jamulus sets a good default value that the operating system should honor.

File: wiki/en/Include-Shared-Commands.md (hunk 2 above)
Comment (ghost) [reply wiki/en/Include-Shared-Commands.md:11]: It looks odd, but it is OK. All that I did was remove a space in each line as seen by the dark pink squares.
Here is what the file looks like:

wiki/en/Include-Shared-Commands.md
```

[comment]: # (This is an include file for use in multiple documents)

- `-h` or `--help`     Display help text         
- `-i` or `--inifile`  Set location of initialization file (overrides default) 
- `-n` or `--nogui`    Disable GUI (for use in headless mode)                      
- `-p` or `--port`     Sets the local UDP port number. Default is 22124 
- `-Q` or `--qos`      Sets the quality of service DS Field byte. Default is 128 (DSCP/CS4). QoS is ignored by Windows. To enable it, see section below.
- `-t` or `--notranslation`   Disable UI language translations
- `-6` or `--enableipv6`      Enable IPv6 addressing (IPv4 is always enabled)
- `-v` or `--version`  Output version information and exit 
```

File: wiki/en/Include-QOS-Windows.md (hunk 1 above)
Comment (ghost) [reply wiki/en/Include-QOS-Windows.md]: Darwin and later kernels were changed to operate in an undocumented? macOS manner.
https://github.com/jamulussoftware/jamulus/discussions/1988#discussion-3551297

Windows did something similar, but left just enough for a workaround.

File: wiki/en/Include-Shared-Commands.md (hunk 2 above)
Comment (ann0see) [reply wiki/en/Include-Shared-Commands.md:11]: Ok. It’s still a bit strange but seems to work. Maybe we should get rid of the spacing all together (but that’s up to another PR and decision)

File: wiki/en/Include-QOS-Windows.md (hunk 3 above)
Comment (ann0see) [reply wiki/en/Include-QOS-Windows.md]: Why do you think links will get broken? Wikipedia should be reliable for example and our link checker should show us that something broke.

Comment (ghost) [reply wiki/en/Include-QOS-Windows.md]: I'm not opposed to adding links, but I decided to mention this because there are many websites nowadays that have so many keywords linked to definitions etc. that it can actually be inconvenient. Feel free to add whatever you like because I won't get in the way.
Linking to anything that is not part of Jamulus documentation will require constant maintenance because it is not under your control. It will also be a mess to package or be moved elsewhere.

**Comment by ann0see** [comment]
@DavidSavinkoff I think this needs a rebase - sorry for that. 
I think we need to get it in soon.

File: wiki/en/Include-QOS-Windows.md (hunk 3 above)
Comment (ghost) [reply wiki/en/Include-QOS-Windows.md]: I simplified the wording by not mentioning buffer bloat and added a link for further information. See new changes to wiki/en/Include-QOS-Windows.md above. You can resolve the conversation if this is acceptable.

**Comment by ghost** [comment]
> @DavidSavinkoff I think this needs a rebase - sorry for that. I think we need to get it in soon.

I don't know what to do for a rebase. BTW I made these changes against the 'release' branch.

**Comment by ann0see** [comment]
Yes, but it should be against next-release since it might need translation

File: wiki/en/Include-QOS-Windows.md [hunk 4]
```diff
@@ -3,7 +3,7 @@
 
 # Quality of Service (QoS)
 
-Jamulus uses DSCP/CS4 to opportunistically deal with buffer bloat. Setting the DS field byte to 128 (or 0x80) in a IPv4 or IPv6 packet header selects DSCP/CS4. Other byte values can be set using the -Q option, e.g. -Q [0..255] (where 0 cripples QoS). If you want to explore the effect of non-default settings, see RFC4594. However, most people will have no need to do this.
+Jamulus uses the DSCP/CS4 quality of service to mitigate network jitter delays. Setting the DS field byte to 128 (or 0x80) in an IPv4 or IPv6 packet header selects DSCP/CS4. Other byte values can be set using the -Q option, e.g. -Q [0..255]. If you want to explore the effect of non-default settings, see [RFC4594](https://datatracker.ietf.org/doc/html/rfc4594) page 18. However, most people will have no need to do this.
```
Comment (pljones) [inline wiki/en/Include-QOS-Windows.md]: > Jamulus uses _the_ DSCP/CS4 quality of service to mitigate network jitter delays.

"the" is syntactically incorrect here.  "DSCP/CS4 quality of service" isn't a thing, it's a concept.

```suggestion
Jamulus uses Quality of Service (QoS) to mitigate network jitter delays.

If you want to explore the effect of non-default settings, see [RFC4594](https://datatracker.ietf.org/doc/html/rfc4594) page 18.  Jamulus sets the DS field byte to 128 (or 0x80) to select DSCP/CS4 in an IPv4 or IPv6 packet header. Other byte values can be set using the -Q option, e.g. -Q [0..255].  However, most people will have no need to do this.
```
